slides/lectures/04/adc/slides.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: section
+---
+# ADC
+Analog to Digital Converter
+
+---
+---
+# Bibliography
+for this section
+
+**Raspberry Pi Ltd**, *[RP2040 Datasheet](https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf)*
+   - Chapter 4 - *Peripherals*
+     - Chapter 4.9 - *ADC and Temperature Sensor*
+       - Subchapter 4.9.1
+       - Subchapter 4.9.2
+       - Subchapter 4.9.5
+
+---
+layout: two-cols
+---
+# ADC
+sampling an analog signal to an array of values
+
+<style>
+.two-columns {
+    grid-template-columns: 3fr 5fr;
+}
+</style>
+
+| | | |
+|-|-|-|
+| *sampling rate* | Hz | the frequency at which a new sample is read |
+| *resolution* | bits | the number of bits used to store a sampled value |
+
+:: right ::
+
+<div align="center">
+<img src="/adc/sampling_values.svg" class="rounded w-150">
+</div>
+
+---
+---
+# Nyquist Theorem
+
+<div grid="~ cols-2 gap-5">
+
+<div>
+
+$$
+sampling_f >= 2 \times max_{f}
+$$
+
+*The **sampling frequency** has to be at least **two times higher** than the **maximum frequency** of the signal.*
```

Comment:
```suggestion
*The **sampling frequency** has to be at least **two times higher** than the **maximum frequency** of the signal to avoid aliasing.*
```